src/main/java/com/elixr/springbootfile/dao/ModelFile.java

```diff
@@ -0,0 +1,34 @@
+package com.elixr.springbootfile.dao;
+
+import lombok.Data;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.FieldType;
+import org.springframework.data.mongodb.core.mapping.MongoId;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.GeneratedValue;
+import javax.validation.constraints.NotEmpty;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+
+@Data
+@Component
+@Document(collection = "files")
+public class ModelFile {
+
+    @MongoId(FieldType.STRING)
```

Comment:
Reason why this is marked as string?

Reply:
just to return the UUID as string format in DB.